packages/flutter_tools/gradle/aar_init_script.gradle

```diff
@@ -121,14 +148,14 @@ projectsEvaluated {
     if (rootProject.property("is-plugin").toBoolean()) {
         assert rootProject.hasProperty("output-dir")
         // In plugin projects, the root project is the plugin.
-        configureProject(rootProject, rootProject.property("output-dir"))
+        configureProject(rootProject, rootProject.property("output-dir"), false)
```

Comment:
For module projects, it should use '--builder-number' as the version number; therefore, I passed the project type to the 'configureProject' function.